CMakeLists.txt

```diff
@@ -325,6 +351,11 @@ IF (BUILD_TESTING)
   function(AddTest) # function for UT
     cmake_parse_arguments(UT "DYN" "TARGET" "LIBS;DYN_LIBS;STATIC_LIBS;SOURCES;DEPENDS" ${ARGN})
     add_executable(${UT_TARGET} ${UT_SOURCES})
+
+    if(ENABLE_COVERAGE)
+      add_coverage(${UT_TARGET})
```

Comment:
Just to clarify, is `add_coverage` defined in `cmakecodecov`?

Reply:
Yes